randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -966,10 +972,18 @@ Extra - room_id: [27]
   * Extra - door_idx: (63,)
   > Pickup (Missile Tank)
       All of the following:
-          # Speedboost to Central Shaft and wrap back around https://youtu.be/jkn5df7GKwo
+          # Speedboost to Central Shaft and wrap back around
```

Comment:
The comment has no reason to say this, especially since it doesn't even explain why. Logically, the only thing thats needed is coming from the left. If you wanna mention/show that it may be slightly easier to kill the desgeega by initially coming from the right and resparking, then do so expllicitly please.